app/models/landing_page/block/document_list.rb

```diff
@@ -1,15 +1,73 @@
 module LandingPage::Block
   class DocumentList < Base
+    attr_reader :taxon_base_path
+
+    TAXON_SEARCH_FIELDS = %w[title
+                             link
+                             description
+                             format
+                             public_timestamp].freeze
+
+    def initialize(block_hash, landing_page)
+      super
+
+      @taxon_base_path = data["taxon_base_path"]
+    end
+
     def full_width?
       false
     end
 
     def items
-      hardcoded_items
+      items_from_taxon.presence || hardcoded_items
```

Comment:
This will return the hardcoded items if there are legitimagely no items in the taxon (i.e. taxon_base_path is configured, the taxon exits, but there are no search results).

Is that what we want?

Reply:
From reading the tests and subsequent commits: yes

I was confused by the term "hardcoded_items" - I'll do a separate comment to explain myself there.

Reply:
Yes, I thought it would be prudent to have a "fallback" option just in case.